### PR TITLE
Fixed #340 Compute sha256sum inside the target directory

### LINKED ISSUE #340: Compute sha256sum inside the target directory
### Situation

Currently, the sha256sum calculation of the assets in the release-droid templates does it in the project root directory. 

```
- name: Generate sha256sum files
  run: find target -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
```

We should first enter into `target/` and then do the sha256sum calculation. 

This solves minor annoyance where users don't have to download jar file into a `target/` folder when checking.

For example,

```
$ sha256sum --check test-db-builder-java-3.3.3.jar.sha256 
sha256sum: target/test-db-builder-java-3.3.3.jar: No such file or directory
target/test-db-builder-java-3.3.3.jar: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
```

For now, users need to create `target/` folder and copy jar there or edit the sha256 file with update path and do the check afterwards.

### Acceptance Criteria

- Release droid upload Github assets workflow is updated

### PR DESCRIPTION
**Pull request by ckunki**
Fixes #340

### CODE REVIEW

File: doc/changes/changes_2.7.0.md [hunk 1]
```diff
@@ -4,11 +4,13 @@ Code name: Enhanced
 
 ## Summary
 
-Interpret regular expressions for excludes in file `.project-keeper.yml` as multiline by default. 
+Interpret regular expressions for excludes in file `.project-keeper.yml` as multiline by default.
 ## Features
 
 * #354: Interpret regular expressions for excludes as multiline by default. <br />
```
Comment (morazow) [inline doc/changes/changes_2.7.0.md:10]:
```suggestion
* #354: Interpreted regular expressions for excludes as multiline by default. <br />
```

File: doc/changes/changes_2.7.0.md [hunk 2]
```diff
@@ -4,11 +4,13 @@ Code name: Enhanced
...
-Some findings of PK have rather long output and contain multiple lines. In order to exclude such findings by regular expressions users needed to use the prefix "(?s)" up to now. PK now provides additional convenience by assuming by default the dot `.` in regular expressions to match any character, including a line terminator.
+Some findings of PK have rather long output and contain multiple lines. In order to exclude such findings by regular expressions users needed to use the prefix `(?s)` up to now. PK now provides additional convenience by assuming by default the dot `.` in regular expressions to match any character, including a line terminator.
+* #340: Compute sha256sum inside the target directory <br />
```
Comment (morazow) [inline doc/changes/changes_2.7.0.md:12]:
```suggestion
* #340: Computed sha256sum inside the target directory <br />
```

Comment (morazow) [reply doc/changes/changes_2.7.0.md:12]: We adopted the past tense in general in change logs some time ago. But this is nit for consistency

File: project-keeper/src/main/resources/templates/.github/workflows/release_droid_upload_github_release_assets.yml [hunk 3]
```diff
@@ -24,7 +24,9 @@ jobs:
       - name: Build with Maven skipping tests
         run: mvn --batch-mode clean verify -DskipTests
       - name: Generate sha256sum files
-        run: find target -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
+        run: |
+          cd target
+          find . -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
```
Comment (morazow) [inline project-keeper/src/main/resources/templates/.github/workflows/release_droid_upload_github_release_assets.yml:29]: :+1:

**Review by morazow — APPROVED** [review]

File: doc/changes/changes_2.7.0.md (hunk 1 above)
Comment (ckunki) [reply doc/changes/changes_2.7.0.md:10]: THX

File: doc/changes/changes_2.7.0.md (hunk 2 above)
Comment (ckunki) [reply doc/changes/changes_2.7.0.md:12]: Thanks - I appreciate accuracy and you eagle eye.

**Pull request merged**